app-frontend/src/app/components/pages/project/createAnalysis/index.js

```diff
@@ -329,6 +334,37 @@ class ProjectCreateAnalysisPageController {
             this.selected.size &&
             this.selected.size === this.layerList.length;
     }
+
+    aoiRequiredModal(layer) {
+        this.modalService
+            .open({
+                component: 'rfFeedbackModal',
+                resolve: {
+                    title: () => `No AOI defined for "${layer.name}"`,
+                    content: () =>`
+                        <h2>
+                            Creating an analyses requires an AOI
+                        </h2>
+                        <p>
+                            but the layer "${layer.name}" does not have an AOI defined.
```

Comment:
```suggestion
                            "${layer.name}" does not have an AOI defined.
```
Subtext reads as a separate sentence due to the difference in font sizes and new line, so this is feels like starting a new sentence with a lower case conjunction.